mysql-test/suite/innodb/t/doublewrite_debug.test

```diff
@@ -57,8 +59,15 @@ set global innodb_saved_page_number_debug = 1;
 set global innodb_fil_make_page_dirty_debug = 0;
 
 set global innodb_buf_flush_list_now = 1;
-let $shutdown_timeout=0;
---source include/shutdown_mysqld.inc
+
+--let CLEANUP_IF_CHECKPOINT=drop table t1, unexpected_checkpoint;
+
+if (`select count(*) from information_schema.system_variables where variable_name='have_sanitizer' and global_value like "MSAN%"`)
+{
+--let CLEANUP_IF_CHECKPOINT=drop table t1;
+}
+
+--source ../include/no_checkpoint_end.inc
```

Comment:
I think that this requires a `#` comment that explains why we are allowing the test to be skipped on the MSAN builder:
```
# Occasionally, a checkpoint would occur on the MSAN builder.
# We do not know the reason, because the failure can only be reproduced if there is
# enough load in that environment.
# Therefore, we allow the test to be skipped when run on MSAN.
# In other environments, we want the test to fail if a checkpoint occurs,
# so that we would catch it if it starts to happen regularly.
```
I got [a skip message](https://buildbot.mariadb.org/#/builders/572/builds/5341/steps/8/logs/stdio) after repeating the MSAN test a few times:
```
innodb.doublewrite_debug '4k,innodb,strict_crc32' w34 [ skipped ]  Extra checkpoint 1 after 67204 (0:10=67204,0:11=67284)
```